apps/meteor/tests/e2e/access-security-page.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { Users } from './fixtures/userStates';
+import { AccountProfile } from './page-objects';
+import { getSettingValueById } from './utils/getSettingValueById';
+import { setSettingValueById } from './utils/setSettingValueById';
+import { test, expect, BaseTestAPI } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe.serial('access-security-page', () => {
+	let e2eEnabledDefaultValue: unknown;
+	let twofaEnabledDefaultValue: unknown;
+	let passwordChangeEnabledDefaultValue: unknown;
+	let poAccountProfile: AccountProfile;
+	let enableSettings: (api: BaseTestAPI, { passwordChange, twofa, e2e }: { passwordChange: boolean; twofa: boolean; e2e: boolean }) => Promise<void>;
+	let isFirstTestOver = false;
+
+	test.beforeAll(async ({ api }) => {
+		[passwordChangeEnabledDefaultValue, twofaEnabledDefaultValue, e2eEnabledDefaultValue] = await Promise.all([
+			getSettingValueById(api, 'Accounts_AllowPasswordChange'),
+			getSettingValueById(api, 'Accounts_TwoFactorAuthentication_Enabled'),
+			getSettingValueById(api, 'E2E_Enable'),
+		]);
+
+		enableSettings = async (api, { passwordChange, twofa, e2e }) => {
+			await Promise.all([
+				setSettingValueById(api, 'Accounts_AllowPasswordChange', passwordChange),
+				setSettingValueById(api, 'Accounts_TwoFactorAuthentication_Enabled', twofa),
+				setSettingValueById(api, 'E2E_Enable', e2e),
+			]);
+		}
```

Comment:
In this case, let's use the `setSettingValueById` directly here instead of creating this function, I think it would be more explicit to see.